src/OTSubscriber.js

```diff
@@ -58,7 +51,7 @@ export default class OTSubscriber extends Component {
 
     this.containerId = uuid();
     const { containerId } = this;
-    const container = document.createElement('div');
+    const container = document.createElement('div', { style: { display: 'flex', flex: 1 } });
```

Comment:
Couldn't you use containerStyle props instead ?

Reply:
ouups yes !